README.md

```diff
@@ -26,69 +26,70 @@ Install the Docker generator:
 npm install -g generator-docker
 ```
 
-Run the generator in the same folder that your project is in:
+## Getting Started
+Once yo docker and VS Code are installed, here's the few steps to debug your first app
+### Node.js
+- From the root of your project, where your server.js file existts, open a terminal window
+```bash
+$ yo docker
+```
+- Answer the prompts, including your image name
+- Open the file you wish to debug and set a breakpoint
+- From VS Code, **[Command]** + **[P]** for the command palate
```

Comment:
fixed palettte
I don't have my windows machine today.  Is command the same for Windows? ⌘P is Mac only